packages/lodestar/test/unit/network/reqresp/rateTracker.test.ts

```diff
@@ -0,0 +1,44 @@
+import {expect} from "chai";
+import sinon from "sinon";
+import {RateTracker} from "../../../../src/network/reqresp/rateTracker";
+
+describe("RateTracker", () => {
+  let rateTracker: RateTracker;
+  const sandbox = sinon.createSandbox();
+
+  beforeEach(() => {
+    rateTracker = new RateTracker({limit: 500, timeoutMs: 60 * 1000});
+    sandbox.useFakeTimers();
+  });
+
+  afterEach(() => {
+    sandbox.restore();
+  });
+
+  it("should request objects up to limit", () => {
+    expect(rateTracker.getRequestedObjectsWithinWindow()).to.be.equal(0);
```

Comment:
Add a comment explaining the idea of the test and what is it simulating, for example: "Simulates a peer doing a fast request of 4 objects in X time, and it should get banned after N etc etc."